app-misc/double-commander/double-commander-1.0.9.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils
+
+DESCRIPTION="Cross Platform file manager."
+HOMEPAGE="http://doublecmd.sourceforge.net/"
+SRC_URI="https://github.com/doublecmd/doublecmd/releases/download/v${PV}/doublecmd-${PV}-src.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+ABBREV="doublecmd"
+
+RESTRICT="strip"
```

Comment:
Why? Because it's Pascal, I guess? Please put a comment saying why.

Reply:
Yes, FreePascal is stripping it as a part of the process. Althought, I will admit, the binary is still gigantic. It's like 40 megs.